pages/common/dotnet-test.md

```diff
@@ -0,0 +1,16 @@
+# dotnet test
+
+> Execute tests for a .NET application.
+> More information: <https://learn.microsoft.com/dotnet/core/tools/dotnet-test>.
+
+- Execute tests for a .NET project / solution in the current directory:
+
+`dotnet test`
+
+- Execute tests for a .NET project / solution in a specific location:
+
+`dotnet test {{path/to/project_or_solution}}`
+
+- Execute tests whose name contains `TestMethod1`:
+
+`dotnet test --filter Name~TestMethod1`
```

Comment:
```suggestion
- Execute tests with a specific filter:

`dotnet test --filter {{Name~TestMethod1}}`
```

Maybe we could generalize this example and also convert the filter to a placeholder.

Reply:
Could I even put a link to the documentation for those filters? It would be this one then: https://learn.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests?pivots=xunit